opds.py

```diff
@@ -1136,15 +1135,6 @@ class LookupAcquisitionFeed(AcquisitionFeed):
     which may be different from the identifier used by the Work's
     default LicensePool.
     """
-    def __init__(self, _db, title, url, works, annotator=None,
-                 messages_by_urn={}, precomposed_entries=[],
-                 require_active_licensepool=True):
-        self.require_active_licensepool=require_active_licensepool
```

Comment:
Since require_active_licensepool is gone there's no need to override the constructor.